project/Deps.scala

```diff
@@ -6,20 +6,20 @@ object Resolvers {
 
 object Deps {
   object V {
-    val algebird = "0.12.2"
+    val algebird = "0.13.0"
     val jackson = "1.9.13"
-    val bijection = "0.7.0"
     val bonsai = "0.2.1"
+    val bijection = "0.9.5"
     val tDigest = "3.1"
 
     val hadoopClient = "2.5.2"
     val scalding = "0.16.1-RC3"
-    val chill = "0.5.2"
+    val chill = "0.9.2"
```

Comment:
this is a bit complex. Different minor versions of chill use different kryo versions, which MAY be an issue if people (foolishly) keep kryo serialized data for the long term. I don't know how Kryo is being used here, so I'm a bit worried. The safest approach is to take the latest `0.7.x` version (0.7.7 I think).